src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -2196,6 +2186,16 @@ private IEnumerable<TypeInferenceInfo> GetCollectionElementType(INamedTypeSymbol
 
                 return SpecializedCollections.EmptyEnumerable<TypeInferenceInfo>();
             }
+
+            private ITypeSymbol GetSymbolReturnType(ISymbol symbol)
```

Comment:
nit: static

Reply:
Turns out there already is a private method for this purpose (and yes, that one IS static :-)). To prevent duplication, I'll use that one.